OpenAPI/kiota/serialization.md

```diff
@@ -188,6 +188,29 @@ const deserializedUser = deserializeFromJson(result, createUserFromDiscriminator
+| number | [double](https://spec.openapis.org/registry/format/double.html) | [double](/dotnet/api/system.double) | [float64](https://pkg.go.dev/builtin#float64) | [Double](https://docs.oracle.com/javase/8/docs/api/java/lang/Double.html) | [number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number) | [float](https://www.php.net/manual/en/language.types.float.php) | [float](https://docs.python.org/3/library/functions.html#float) | Yes |
+| number | [decimal](https://spec.openapis.org/registry/format/decimal.html) | [decimal](/dotnet/api/system.decimal) | [float64](https://pkg.go.dev/builtin#float64) | [BigDecimal](https://docs.oracle.com/javase/8/docs/api/java/math/BigDecimal.html) | [number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number) | [float](https://www.php.net/manual/en/language.types.float.php) | [float](https://docs.python.org/3/library/functions.html#float) | |
+| string | N/A | [string](/dotnet/api/system.string) | [string](https://pkg.go.dev/builtin#string) | [String](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html) | [string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) | [string](https://www.php.net/manual/en/language.types.string.php) | [string](https://docs.python.org/3/library/string.html) | Yes |
+| string | [uuid](https://spec.openapis.org/registry/format/uuid.html) | [Guid](/dotnet/api/system.guid) | [UUID](https://pkg.go.dev/github.com/google/uuid#UUID) | [UUID](https://docs.oracle.com/javase/8/docs/api/java/util/UUID.html) | [Guid](https://www.npmjs.com/package/guid-typescript) | [string](https://www.php.net/manual/en/language.types.string.php) | [UUID](https://docs.python.org/3/library/uuid.html) | |
+| string | [date-time](https://spec.openapis.org/registry/format/date-time.html) | [DateTimeOffset](/dotnet/api/system.datetimeoffset) | [time](https://pkg.go.dev/time#Time) | [OffsetDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/OffsetDateTime.html) | [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) | [DateTime](https://www.php.net/manual/en/class.datetime.php) | [datetime](https://docs.python.org/3/library/datetime.html) | |
+| string | [date](https://spec.openapis.org/registry/format/date.html) | [Kiota Date](https://github.com/microsoft/kiota-dotnet/blob/main/src/abstractions/Date.cs) | [Kiota Date Only](https://github.com/microsoft/kiota-abstractions-go/blob/main/serialization/date_only.go) | [LocalDate](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html) | [Kiota Date Only](https://github.com/microsoft/kiota-typescript/blob/main/packages/abstractions/src/dateOnly.ts) | [Kiota Date](https://github.com/microsoft/kiota-abstractions-php/blob/main/src/Types/Date.php) | [date](https://docs.python.org/3/library/datetime.html) | |
```

Comment:
```suggestion
| string | [date](https://spec.openapis.org/registry/format/date.html) | [Kiota Date/DateOnly](https://github.com/microsoft/kiota-dotnet/blob/main/src/abstractions/Date.cs) | [Kiota Date Only](https://github.com/microsoft/kiota-abstractions-go/blob/main/serialization/date_only.go) | [LocalDate](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html) | [Kiota Date Only](https://github.com/microsoft/kiota-typescript/blob/main/packages/abstractions/src/dateOnly.ts) | [Kiota Date](https://github.com/microsoft/kiota-abstractions-php/blob/main/src/Types/Date.php) | [date](https://docs.python.org/3/library/datetime.html) | |
```

Could we make the argument that this is mapped to `DateOnly` too with the work done to use `implicit operator` in C# for net6 targets?

Reply:
I'm going to add it as a note below so it's clear for all types (DateOnly, TimeOnly)

Reply:
added